### ISSUE TITLE
Spurious warnings in copts

### ISSUE THREAD

**Issue by jayconrod** [bazelbuild/rules_go#702]
Suppose we have a cgo file with some flags:

```go
package foo

/*
#cgo CFLAGS: -DFOO -DBAR
*/
import "C"
```

Gazelle will generate a rule like this:

```bzl
go_library(
    name = "go_default_library",
    srcs = ["foo.go"],
    cgo = True,
    copts = ["-DFOO -DBAR"],
    visibility = ["//visibility:public"],
)
```

Bazel will emit warnings like this:

```
WARNING: /usr/local/google/home/jayconrod/Code/scratch/BUILD.bazel:7:1: in copts attribute of cc_library rule //:go_default_library.cgo_c_lib: each item in the list should contain only one option. Since this rule was created by the macro 'go_library_macro', the error might have been caused by the macro implementation in /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/1df8d8ad4cb35ee83522d81953b4d69f/external/io_bazel_rules_go/go/private/cgo.bzl:298:15.
WARNING: /usr/local/google/home/jayconrod/Code/scratch/BUILD.bazel:7:1: in copts attribute of cc_binary rule //:go_default_library._cgo_.o: each item in the list should contain only one option. Since this rule was created by the macro 'go_library_macro', the error might have been caused by the macro implementation in /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/1df8d8ad4cb35ee83522d81953b4d69f/external/io_bazel_rules_go/go/private/cgo.bzl:319:15.
```

Since #652, Gazelle keeps options from `#cgo` directives in single strings, like `"-DFOO -DBAR"` above. This was needed to fix #646: in Darwin, `-framework` options in `clinkopts` need to be passed as a single string, since Bazel performs label expansion on strings that don't start with '-'. Bazel performs Bourne shell tokenization, so it's safe to pass multiple arguments in the same string.

Unfortunately, in `copts`, Bazel emits the warning above. It's not strictly necessary to pass multiple arguments in the same string for `copts`, since Bazel does not perform label expansion, and I'm not sure if -framework arguments can appear there anyway. However, it would be nice to process CFLAGS and LDFLAGS the same way.

**Comment by jayconrod** [comment]
Filed bazelbuild/bazel#3526 to remove the warning.

**Comment by jayconrod** [comment]
Fixed upstream
